dv/uvm/core_ibex/common/ibex_cosim_agent/spike_cosim_dpi.cc

```diff
@@ -23,6 +23,7 @@ void *spike_cosim_init(const char *isa_string, svBitVecVal *start_pc,
   SpikeCosim *cosim = new SpikeCosim(isa_string, start_pc[0], start_mtvec[0],
                                      log_file_path, false, true);
   cosim->add_memory(0x80000000, 0x80000000);
+  cosim->add_memory(0x00000000, 0x80000000);
```

Comment:
I don't think this is correct. This would cause any memory address from `0x8000_0000` to `0xffff_ffff` to be out of bounds right?

Reply:
I thought it would create two separate memories. I tested this and haven't seen the behaviour you suggested.

Reply:
Ah yes, I misunderstood. Should work!